tasks/request_files/request_files.py

```diff
@@ -394,8 +400,8 @@ def process_granule(
         max_retries: todo
         retry_sleep_secs: todo
         retrieval_type: todo
-        db_queue_url: todo
         job_id: The unique identifier used for tracking requests.
+        db_queue_url: todo
```

Comment:
Replace todo with information on what it is

Reply:
Fixed up todos